sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -51,34 +53,31 @@
 
     private List<URI> contactedReplicas;
     private Set<URI> failedReplicas;
-    private ZonedDateTime requestStartTime;
-    private ZonedDateTime requestEndTime;
+    private ZonedDateTime requestStartTimeUTC;
+    private ZonedDateTime requestEndTimeUTC;
     private Set<URI> regionsContacted;
     private RetryContext retryContext;
     private GatewayStatistics gatewayStatistics;
     private RequestTimeline transportRequestTimeline;
+    private MetadataDiagnosticsContext metadataDiagnosticsContext;
+    private SerializationDiagnosticsContext serializationDiagnosticsContext;
 
     ClientSideRequestStatistics() {
-        this.requestStartTime = ZonedDateTime.now(ZoneOffset.UTC);
-        this.requestEndTime = ZonedDateTime.now(ZoneOffset.UTC);
+        this.requestStartTimeUTC = ZonedDateTime.now(ZoneOffset.UTC);
```

Comment:
InstantVsZonedDateTime.instantTime       thrpt    5  323658648.596 ± 21659882.776  ops/s
InstantVsZonedDateTime.zonedDateTime     thrpt    5  126614135.659 ±  5709318.981  ops/s
InstantVsZonedDateTime.zonedDateTimeUTC  thrpt    5  124846883.459 ± 16047362.965  ops/s

As per the above number instantTime is performing good.
How ever I ran the regular ReadThroughput benchmark with and without PR , results are same, so in overall end to end test it is not making noticeable difference.
However it make sense to use instantTime  if we want UTC . So we have three ways from here onward
1. Change all the zonedDateTimeUTC   to instantTime  throughout the code(at least in all diagnostics in a separate PR soon after this) and merge this PR
2. Changed  zonedDateTimeUTC  to instantTime only in the scope of this PR
3. Changed  all the zonedDateTimeUTC   to instantTime with this PR itself .

I am inclined toward 1 as it will be cleaner and more modular.

Reply:
@simplynaveen20 Let's do 1 as a separate PR - we should change ZonedDateTimeUTC to instantTime. There is no need for ZonedDateTimeUTC in SDK, as everything should be in UTC.

Reply:
option 1 makes sense.